build/build-bundle.js

```diff
@@ -255,5 +256,5 @@ if (esMain(import.meta)) {
 }
 
 export {
-  buildBundle,
+  buildLrBundle,
```

Comment:
?

Reply:
Perhaps at some point this build-bundle file was used for a few different bundles, but these days it's only used by build-lightrider-bundles. So.. it'd be good to remind ourselves that this is exclusive to the LR build